doc/Makefile

```
@@ -27,4 +27,4 @@ clean:
 pdf:
 	@$(SPHINXBUILD) -M latex "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 	cd build/latex && latexmk -r latexmkrc -pdf *.tex -interaction=nonstopmode || true
-	(test -f build/latex/pyansys*.pdf && echo pdf exists) || exit 1
\ No newline at end of file
+	(test -f build/latex/ansys_sphinx_theme.pdf && echo pdf exists) || exit 1
```

There's no reason to remove `*`. 

```suggestion
	(test -f build/latex/ansys*.pdf && echo pdf exists) || exit 1
```

---

@akaszynski For pdf cover we are using the images in the form of pdf, named ansys_logo_white.pdf and ansys_logo_white_cropped.pdf. so while building it was showing error.